src/App.tsx

```diff
@@ -1,9 +1,25 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [movieName, setMovieName] = useState('');
+
+  const handleOnChange = (event:React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = event.target;
+
+    setMovieName(value);
+  };
+
+  const visibleMoviesOnList = moviesFromServer.filter(movie => {
+    const title = movie.title.toLowerCase();
+    const description = movie.description.toLowerCase();
+    const name = movieName.toLowerCase().trim();
```

Comment:
You'd better rename this variable to something like 'preparedName' for better understanding what exactly name you mean)

Reply:
Thanks! Just fix it)